### PR TITLE
Feature/os 154 UI refactoring

### PR DESCRIPTION
**Pull request by Avedis92**

### CODE REVIEW

File: os-ui/src/component/Button/Components/SignIn-Button.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import styles from '../Button-Styles.module.scss';
+
+/* eslint-disable max-len */
+
+export const SignInButton = ({ children, className, onClick }:{children:string, className:Array<string>, onClick:()=>void}) => {
```
Comment (SevakMart) [inline os-ui/src/component/Button/Components/SignIn-Button.tsx:5]: why do we need different buttons if they are similar ?

File: os-ui/src/component/LearningPath/LearningPath.tsx [hunk 2]
```diff
@@ -1,51 +1,53 @@
...
+      <div className={mainBody} onClick={headToCourseDescriptionPage}>
+        <div className={ratingContent}>
+          <p className={iconStyle}><StarIcon /></p>
+          <p data-testid={ratingValue} className={ratingValue}>{courseInfo.rating}</p>
+        </div>
+        <p data-testid={courseInfo.title} className={courseTitle}>{courseInfo.title}</p>
+        <div className={keywordsContent}>
+          <div className={keyword}>{courseInfo.keywords[0]}</div>
+          <div className={keyword}>{courseInfo.keywords[1]}</div>
+          {courseInfo.keywords.length > 2 && (
```
Comment (SevakMart) [inline os-ui/src/component/LearningPath/LearningPath.tsx:43]: you can make '2' a constant

File: os-ui/src/component/Navbar-Component/NavbarOnSignIn/NavbarOnSignIn.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+  return (
+    <nav className={mainContent}>
+      <h2 onClick={() => navigate('/')}>{t('string.homePage.navBar.logo')}</h2>
+      <div className={navMainContent}>
+        <p onClick={() => navigate('/exploreLearningPaths')}>{t('string.homePage.navBar.allLearningPaths')}</p>
+        <p onClick={() => navigate('/myLearningPath')}>{t('string.homePage.navBar.myLearningPaths')}</p>
+        <p onClick={() => navigate('/mentors')}>{t('string.homePage.navBar.mentors')}</p>
+        <div className={bookmarkIconStyle}>
+          <BookmarkIcon iconSize="20px" />
```
Comment (SevakMart) [inline os-ui/src/component/Navbar-Component/NavbarOnSignIn/NavbarOnSignIn.tsx:34]: is it ok to hardcode the size in pixels ?

Comment (Avedis92) [reply os-ui/src/component/Navbar-Component/NavbarOnSignIn/NavbarOnSignIn.tsx:34]: iconSize is a prop which is the value of the font size of style attribute inside the BookmarkIcon component

File: os-ui/src/component/LearningPath/LearningPath.tsx (hunk 2 above)
Comment (Avedis92) [reply os-ui/src/component/LearningPath/LearningPath.tsx:43]: ok

**Review by SevakMart — APPROVED** [review]

**Pull request merged**